.ci/eslint-plugin-zotero-translator/lib/rules/test-cases.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const translators = require('../translators').cache;
+
+module.exports = {
+	meta: {
+		type: 'problem',
+
+		docs: {
+			description: 'disallow invalid test input',
+			category: 'Possible Errors',
+		},
+	},
+
+	create: function (context) {
+		return {
+			Program: function (node) {
+				const translator = translators.get(context.getFilename());
+
+				const declaration = node.body.find(node => node.type === 'VariableDeclaration' && node.declarations.length === 1 && node.declarations[0].id.name === 'testCases');
+				const testCases = declaration && declaration.declarations[0].init && declaration.declarations[0].init.type === 'ArrayExpression' ? declaration.declarations[0].init.elements : [];
```

Comment:
Split up lines

e.g.,

```
const testCases = declaration
    && declaration.declarations[0].init
    && declaration.declarations[0].init.type === 'ArrayExpression'
        ? declaration.declarations[0].init.elements
        : [];
```

Reply:
If I indent that way I run into `Expected indentation of 5 tabs but found 6`

Reply:
We can look into that later. That style should be allowed, and is preferable to a very long line.

Reply:
So just to make sure: leave indented as above, and leave the two lint errors in place?

Reply:
Yes please.

Reply:
Done.